From b0424f7ecd079a7a8ce66d793efe551815e53929 Mon Sep 17 00:00:00 2001
From: "Tekriwal, Tanuj" <tanuj.tekriwal@intel.corp-partner.google.com>
Date: Mon, 8 Jun 2026 12:26:27 +0000
Subject: [PATCH] Adding missing dep for latest a16 code

Tracked-On: NA
Signed-off-by: Tekriwal, Tanuj <tanuj.tekriwal@intel.corp-partner.google.com>
---
 Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Android.bp b/Android.bp
index 5113e39..eb6e49b 100644
--- a/Android.bp
+++ b/Android.bp
@@ -100,6 +100,7 @@ bootstrap_go_package {
     name: "mfx_c2",
     pkgPath: "android/soong/vendor/intel/mfx_c2",
     deps: [
+        "soong-android",
         "soong-cc",
     ],
     srcs: [
-- 
2.34.1

